content/en/docs/concepts/configuration/secret.md

```diff
@@ -24,6 +24,13 @@ a password, a token, or a key. Such information might otherwise be put in a
 Pod specification or in an image. Users can create Secrets and the system
 also creates some Secrets.
 
+Kubernetes Secrets are, by default, stored as unencrypted Base64 strings. Once written
+to Kubernetes, they can be retrieved - as plain text - by anyone with API access. In order
```

Comment:
```suggestion
to Kubernetes, they can be retrieved - as plain text - by anyone with API access, **or by anyone with access to Kubernetes' underlying data store, etcd**. In order
```

Reply:
If someone has enabled [encryption at rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/) - which I hope they would - and are protecting the encryption keys appropriately, then access to etcd _doesn't_ get you access to Secrets in a useful form.

Reply:
While true, that's not the _default_ behavior

Reply:
Let's leave readers aware of both the risks of the defaults and the opportunities to address that. It's only correct to document that “anyone with access to Kubernetes' underlying data store” can read Secrets if we can qualify that statement.

Reply:
Updated suggestion.

Reply:
Thanks @sethvargo